import-export-cli/git/gitUtils.go

```diff
@@ -388,9 +388,12 @@ func handleIfError(err error, failedProjects map[string][]*params.ProjectParams,
 // updatedProjectsPerType is a map of string -> ProjectParams which consists of updated projects per each type (API, App..)
 // Returns bool, true if any deleted projects exists so the process should continue with project deletion path
 // Returns map[string][]*params.ProjectParams, a map of project type (API, App.. ) to each project detail which are
-//  deleted projects
+//
```

Comment:
Seems this change is not required. Please check.

Reply:
Resolved by https://github.com/wso2/product-apim-tooling/pull/1027/commits/bf810875b56eb0bb7f3328899408af14d8d243e8